packages/kit/src/ignore.ts

```diff
@@ -1,7 +1,9 @@
 import { existsSync, readFileSync } from 'node:fs'
 import ignore from 'ignore'
 import { join, relative, resolve } from 'pathe'
-import { tryUseNuxt } from './context'
+import { tryUseNuxt, useNuxt } from './context'
+
+const checkIgnoreOutdated = (nuxt = useNuxt()) => nuxt._ignorePatterns?.join(',') !== nuxt.options.ignore.join(',')
```

Comment:
This will always return false, right? It doesn't resolve the patterns (e.g. `.{js,mjs}`).

Reply:
Oh, you're quite right; forgot to consider that - good spot! Fixed 👍